scripts/model/portList.js

```diff
@@ -1,58 +1,84 @@
 /*portList.js this file controls population of the portfolio list as well as functionality of the portfolio item article preview*/
 /*resource templating logic*/
+(function(module){
+  /*constructor for PortListItem object*/
+  function PortListItem (options){
```

Comment:
Can you combine PortListItem and Resource into a single object type?

With some small exceptions, they share structure and methods.